Source: maxice8
Upstream: no
Reason: restores versioning of libXvMC drivers that was on autotools

--- src/gallium/targets/xvmc/meson.build
+++ src/gallium/targets/xvmc/meson.build
@@ -51,7 +51,7 @@ libxvmc_gallium = shared_library(
 
 foreach d : [[with_gallium_r600, 'r600'], [with_gallium_nouveau, 'nouveau']]
   if d[0]
-    xvmc_drivers += 'libXvMC@0@.so'.format(d[1])
+    xvmc_drivers += 'libXvMC@0@.so.1.0.0'.format(d[1])
   endif
 endforeach
 
